source/getting-started.txt

```diff
@@ -0,0 +1,72 @@
+.. _relational-migrator-getting-started:
+
+===============
+Getting Started
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Relational Migrator allows you to bring your relational workloads 
+to MongoDB. 
+
+To get familiar with Relational Migrator capabilities, 
```

Comment:
(info typing -- before you begin)
Two options here. If you think it's critical that folks review this info before they get started, you can add as a "Before you Begin" section. Otherwise, consider moving these links to a bottom Learn More section.